Editors/RoomEditor.ui

```diff
@@ -1306,7 +1307,7 @@
             <x>0</x>
             <y>0</y>
             <width>712</width>
-            <height>572</height>
+            <height>571</height>
```

Comment:
?

Reply:
Umm not sure, it was automatic and I'm not sure how to fix that in ui editor.

Reply:
Fixed, manually edited `RoomEditor.ui`